SECURITY_CONTACTS

```diff
@@ -10,6 +10,5 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-bentheelder
 krzyzacy
 rmmh
```

Comment:
(maybe put rest of oncalls here and remove Ryan? I'm not sure if I want to solo here :joy: )

Reply:
dropped ryan and added an @spiffxp todo to add more